README.md

```diff
@@ -61,39 +62,39 @@ func main() {
 
 Requirements:
 
-* Hazelcast Go client is compatible only with Hazelcast IMDG 4.x and above.
-* We support Go 1.15 and up.
+* Hazelcast Go client is compatible only with Hazelcast 4.x and 5.x.
```

Comment:
Don't we need to add 6.x here once it is released 😀 ? Maybe 4.x and above is better.

Reply:
6.x is a long way :) I specifically wanted to mention 5.x.